src/main.cpp

```diff
@@ -163,6 +169,24 @@ int main ( int argc, char** argv )
             continue;
         }
 
+        // JSON-RPC port number ------------------------------------------------
+        if ( GetNumericArgument ( argc, argv, i, "--jsonrpcport", "--jsonrpcport", 0, 65535, rDbleArgument ) )
```

Comment:
Personally, I'd rather reserve the short arguments for things _most_ people will use, rather than just use them up on a first-come-first-served basis.

Reply:
Good point.